index.d.ts

```diff
@@ -0,0 +1,9 @@
+type ClassValue = string | number | ClassDictionary | ClassArray;
+
+interface ClassDictionary {
+	[id: string]: boolean;
+}
+
+interface ClassArray extends Array<ClassValue> { }
+
+export default function(...classes: ClassValue[]): string;
```

Comment:
Yeah, updating the DT definition sounds like a good idea now that the repo has switched over to `@types` completely. Once a release of classnames is made that includes definitions though, we should remove those typings from DT (and update notNeededPackages.json in the same PR).